deliverables/README.md

```diff
@@ -21,48 +21,117 @@ For projects on Code Academy, or other external exercises, please just paste a l
+
 
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
-Why do we separate HTML and CSS?  
+Why do we separate HTML and CSS?
+You can do most CSS in the HTML (in line CSS), but it is easier to visualize and organize code if they are separated into "mark up" and "make up". Also by using a separate CSS file, you can have more complex and specifc instructions so there is less work. 
+
+
 
 #### HTML
-What are classes and IDs (and how are they different)?  
+What are classes and IDs (and how are they different)?
+Classes are groupings of content that you want the same CSS changes to be applied to. If you create (li class"fancy") this line along with all the other members of that class, will be treated the same. 
+An ID applies to a single element that you want to be different from the of the same element. If you only wanted one line to have a partticular style, you can use the ID attribute. 
+Classes and IDs give more specific instructions and functions compared to regular tags.
+
 What are elements?  
+Elements are the content of a particular tag. Ex. 'this is a paragraph', the text (this is a paragraph) is an element while the p are the tags. 
+
 What are tags?  
-What are attributes?  
-What are forms?  
-What is a div?  
+Tags give specification to elements on where they will be in the code. Tags open with <> and close with </>. A tag of h3 blah blah h3 will be in a different place and have a different meaning than the same text with p tags. 
+
+What are attributes?  Attributes are properties of code that assign a different meaning to a common tag. Ex. p id"serious". The ID makes the paragraph differnt than a normal paragraph.
+
+What are forms? 
+Forms are special elements called controls that are used to make things like check boxes, menus etc. 
+
+What is a div?
+Divs are containers that allow the code to be put into sections on the page. 
+
+What are selectors? 
+Selectors use attributes of an HTML element to select a specific element. Ex: # is a selector indicating ID in CSS.
 
-What are selectors?  
 What are properties?  
+Once an attribute is assigned and the browser processes the code, a node will be created and these nodes have properties. Properties like align, alt, attributes etc. 
+
 What are values?  
-How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
+Values give a hexadecimal value to colors. Instead of red, there is #458393. It gives a value to colors. 
+
+How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?
+The default styles are inherited, ie passed down to a younger generation, from the parent element to the child element. 
+
 What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+You can use text-align or through commands like center me.
```

Comment:
text-align isn't going to move an element; it will only move its contents.  Examples would be:  

`float`, `display`, `position`, `margin`, `top`, `left`, etc